xml/System.Windows.Forms/TextBox.xml

```diff
@@ -948,8 +948,8 @@
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the text that is displayed when the control has no text and is not on focus.</summary>
```

Comment:
```suggestion
        <summary>Gets or sets the text that is displayed when the control has no text and is not have the focus.</summary>
```

Reply:
@rpetrusha your suggestion has a typo. I think you meant this:

```suggestion
        <summary>Gets or sets the text that is displayed when the control has no text and does not have the focus.</summary>
```